src/main/java/net/greatstart/validators/NewUserValidator.java

```diff
@@ -0,0 +1,25 @@
+package net.greatstart.validators;
+
+import net.greatstart.dto.DtoUser;
+import net.greatstart.services.UserService;
+import org.springframework.validation.Errors;
+
+public class NewUserValidator {
```

Comment:
I would prefer `UserValidationService` or something similar - because this is service layer